nubis/puppet/files/nubis_users/nubis-users.toml

```diff
@@ -0,0 +1,12 @@
+[template]
+src = "nubis-users.tmpl"
+dest = "/etc/puppet/yaml/nubis_users/common.yaml"
+prefix = "/nubis-users"
+
+keys = [
```

Comment:
These keys should be protected by the platform ACL, to make that easy, I would move these keys under /environment/<env>/user-managmenent/{users,sudo-users,global-admins} as it's already protected and leave the top-level keyspace free for the applications to use